### PR TITLE
Fix retrieval of pdfsets from hepforge repo

### PR DESCRIPTION
**Pull request by aalkin**
Hepforge does not add Content-Length to the response header which creates an exception in the code that relies on this field to calculate file size. The simplest solution is to comment the relevant lines, however hepforge repository may change behavior again in the future.

### CODE REVIEW

File: bin/lhapdf.in [hunk 1]
```diff
@@ -277,16 +277,16 @@ def download_url(source, dest_dir, dryrun=False):
             import urllib2 as urllib
         try:
             u = urllib.urlopen(url)
-            file_size = int(u.info().get('Content-Length')[0])
+            #file_size = int(u.info().get('Content-Length')[0])
```
Comment (qgp) [inline bin/lhapdf.in]:
```suggestion
            file_size = int(u.info().get('Content-Length', (-1,))[0])
```
wouldn't that be enough and avoid commenting out the other lines (which I'd avoid if possible to keep changes to be ported to a future version to a minimum)

**Comment by qgp** [comment]
Changed this PR to be against alice/v6.2.1, which is the branches containing the version with ALICE patches.

File: bin/lhapdf.in [hunk 2]
```diff
@@ -286,7 +286,7 @@ def download_url(source, dest_dir, dryrun=False):
         logging.debug('Downloading from %s' % url)
         logging.debug('Downloading to %s' % dest_filepath)
         if dryrun:
-            logging.info('%s [%s]' % (os.path.basename(url), convertBytes(file_size)))
+            #logging.info('%s [%s]' % (os.path.basename(url), convertBytes(file_size)))
```
Comment (qgp) [inline bin/lhapdf.in]: Does this still have to be commented out? The default value of -1 should make this work.

File: bin/lhapdf.in [hunk 3]
```diff
@@ -306,7 +306,7 @@ def download_url(source, dest_dir, dryrun=False):
                     dest_file.write(buffer)
 
                     status  = chr(13) + '%s: ' % os.path.basename(url)
-                    status += r'%s [%3.1f%%]' % (convertBytes(file_size_dl).rjust(10), file_size_dl * 100. / file_size)
+                    #status += r'%s [%3.1f%%]' % (convertBytes(file_size_dl).rjust(10), file_size_dl * 100. / file_size)
                     sys.stdout.write(status+' ')
```
Comment (qgp) [inline bin/lhapdf.in]: Does this still have to be commented out? See above.

Comment (aalkin) [reply bin/lhapdf.in]: Good point. I've uncommented the lines that use "file_size" variable.

**Pull request merged**